task/fbc-validation/0.1/README.md

```diff
@@ -1,11 +1,9 @@
 # fbc-validation task
 
 ## Description:
-The fbc-validation task will ensure FBC (File based catalog) components uniquely linted to ensure they're properly
-constructed as part of the build pipeline. To validate the image in build pipeline, Skopeo is used to extract
-information from the image itself and then contents are checked using the OpenShift Operator Framework.  The binary
-used to run the validation is extracted from the base image for the component being tested.  Because of this, the
-base image must come from a trusted source.  Trusted sources are declared in `ALLOWED_BASE_IMAGES` in fbc-validation.yaml.
+Ensures file-based catalog (FBC) components are uniquely linted for proper construction as part of build pipeline.
+For details on specific checks, see the USAGE.md file.
+For troubleshooting assistance, see the TROUBLESHOOTING.md file.
```

Comment:
Can we auto-generate this? Something like
```
if [ -f $(dirname $RESOURCE)/USAGE.md ]; then
  echo
  echo -n "For further information on how to use the task, see the USAGE.md file."
fi
if [ -f $(dirname $RESOURCE)/TROUBLESHOOTING.md ]; then
  echo
  echo -n "For troubleshooting assistance, see the TROUBLESHOOTING.md file."
fi
if [ -f $(dirname $RESOURCE)/USAGE.md ] | [ -f $(dirname $RESOURCE)/TROUBLESHOOTING.md ]; then
  echo
fi
```
between these lines: https://github.com/konflux-ci/build-definitions/blob/main/hack/generate-readme.sh#L10-L11 ?

Reply:
You don't want me to include this information [here](https://github.com/konflux-ci/build-definitions/pull/1327/files#diff-b43bfce7caedfc108e2678a44c111a52a34313d0a2cc255f8a835df82b8610f0R5-R6)?
As part of `task.spec.description`, this information would be populated here already. 

I wasn't sure if folks already had easy access to the markdown files, so I was trying to ensure that the task itself held the necessary references.

Reply:
Oh, I see that you put it in the task description itself. I didn't notice that before. I think that we would eventually like to render the build-definitions makdown files in konflux-ci.dev/docs. If we establish the pattern in the markdown doc construction then we wouldn't have to include the repeated information in every task description.

The primary desire for the change was to prevent the README from getting overwritten. That won't happen with your change, so this is now a more minot nitpick.

Reply:
I added it to the automation, but stopped short of regenerating the README.md file from it.  There are changes that I lack context to accept/refuse there.  In addition to whitespace changes which only matter to folks viewing the raw content, it also drops the 'Source repository...' and ['Additional links'](https://github.com/konflux-ci/build-definitions/pull/1327/files#diff-b43bfce7caedfc108e2678a44c111a52a34313d0a2cc255f8a835df82b8610f0L27-L30) sections.